keys/signer_pallas_test.go

```diff
@@ -28,31 +27,27 @@ var keypair *KeyPair
 var txnBytes []byte
 
 func init() {
-	keypair, _ = GenerateKeypair(types.Pallas)
+	keypair, _ = ImportPrivateKey(
```

Comment:
Can we add more test cases for covering edge cases since there are lot of `if` loops while executing `constructTransaction`